### PR TITLE
search and movie

### PR DESCRIPTION
**Pull request by vacrozet**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -33,11 +34,12 @@ class Index extends React.Component {
         <Switch>
           <Route exact path='/login' component={Login} />
           <Route exact path='/register' component={Register} />
-          <Route exact path='/accueil' component={Popular} />
-          <Route exact path='/top_rated' component={TopRated} />
+          <Route exact path='/accueil' component={Popular} match={this.props.match} />
```
Comment (Its-Alex) [inline src/index.js]: Not work... Need to do like this
```javascript
component={({ match, location, history }) => 
  <Popular match={match} />
}
```

File: src/index.js [hunk 2]
```diff
@@ -33,11 +34,12 @@ class Index extends React.Component {
...
+          <Route exact path='/top_rated' component={TopRated} match={this.props.match} />
```
Comment (Its-Alex) [inline src/index.js]: Same here

File: src/index.js [hunk 3]
```diff
@@ -33,11 +34,12 @@ class Index extends React.Component {
...
           <Route exact path='/profile' component={Profile} />
           <Route exact path='/profile/:id' component={OtherProfile} />
-          <Route exact path='/movie/:id' component={Movie} />
+          <Route exact path='/movie/:id' component={Movie} match={this.props.match} />
```
Comment (Its-Alex) [inline src/index.js]: Same here

File: src/index.js [hunk 4]
```diff
@@ -33,11 +34,12 @@ class Index extends React.Component {
...
+          <Route exact path='/search/:id' component={Search} match={this.props.match} />
```
Comment (Its-Alex) [inline src/index.js]: Same here

**Review by Its-Alex — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (vacrozet) [reply src/index.js]: Expected an assignment or function call and instead saw an expression. (no-unused-expressions)

Comment (vacrozet) [reply src/index.js]: Mister "I know everything" !!

Comment (Its-Alex) [reply src/index.js]: ... Plus teubé que toi c'est rare

Comment (Its-Alex) [reply src/index.js]: J'te laisse chercher

**Review by Its-Alex — APPROVED** [review]

**Pull request merged**